### ISSUE TITLE
[V6 RC3] FieldArray examples don't work

### ISSUE THREAD

**Issue by Sir4ur0n** [erikras/redux-form#1433]
I just tested the V6 RC3 example on FieldArray, and it didn't work out of the box for me.

I am talking about these pages:
http://redux-form.com/6.0.0-rc.3/docs/MigrationGuide.md/ (FieldArray section)
http://redux-form.com/6.0.0-rc.3/examples/fieldArrays/

The example shows:

```
render() {
  return (
    <div>
      <FieldArray name="awards" component={awards =>
        <div>
          <ul>
            {awards.map((name, index) => <li key={index}>
              <label>Award #{index + 1}</label>
              <Field name={name} type="text" component="input"/>
            </li>)}
          </ul>
          <button onClick={() => awards.push()}>Add Award</button>
        </div>
      }/>
    </div>
  )
}
```

But when I use this syntax, I get an error:

> Uncaught TypeError: awards.map is not a function

I have a similar problem with forEach.

After debugging I found "map" and "forEach" functions were available not in "awards", but in "awards.fields", thus I had to change my code to the following:

```
render() {
  return (
    <div>
      <FieldArray name="awards" component={awards =>
        <div>
          <ul>
            {awards.fields.map((name, index) => <li key={index}> // <--- Change here
              <label>Award #{index + 1}</label>
              <Field name={name} type="text" component="input"/>
            </li>)}
          </ul>
          <button onClick={() => awards.push()}>Add Award</button>
        </div>
      }/>
    </div>
  )
}
```

And now it works.
Did I do something wrong, or is there a mistake in both examples?

**Comment by frankjwu** [comment]
+1 @Sir4ur0n, same thing here using V6 RC3

**Comment by gustavohenke** [comment]
That was a problem with the docs. `awards` used to be what we usually call... `props`!
It was fixed in e21380772781257383ccbdf24dd4c49e1f94092b
